libevm/stateconf/conf.go

```diff
@@ -17,7 +17,52 @@
 // Package stateconf configures state management.
 package stateconf
 
-import "github.com/ava-labs/libevm/libevm/options"
+import (
+	"github.com/ava-labs/libevm/libevm/options"
+)
+
+// A StateDBCommitOption configures the behaviour of all update calls within the
+// state.StateDB.Commit() implementations.
+// This is provided in two distinct types to allow customizability to both
+// snapshot and trie database updates, but are ignored in libevm implementations.
```

Comment:
```suggestion
// A StateDBCommitOption configures the behaviour of state.StateDB.Commit().
```

While what you've said in the rest of the comment is true now, it may not always be the case. The details can be inferred from the functions that create these options, and will be obvious when viewing the formatted documentation. Try running `godoc` within the `libevm` repo and then navigating to `localhost:6060` and finding the `stateconf` package.